style.css

```diff
@@ -0,0 +1,269 @@
+* {
+  font-family: "Roboto", Tahoma, Helvetica, Arial, sans-serif;
+  box-sizing: border-box;
+}
+
+.title1 {
+  margin-bottom: 40px;
+  padding-bottom: 25px;
+}
+
+h1 {
+  left: 180px;
+
+  position: absolute;
+  color: #c9f364;
+  text-align: center;
+  margin-top: 1px;
+}
+
+/* LOGO */
+
+.title1 > .logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  margin-top: 1px;
+  margin-left: 40px;
+
+  background: linear-gradient(90deg, #459428 50%, rgba(9, 9, 121, 1) 50%);
+  z-index: 100;
+}
+
+.first,
+.lijevo,
+.sredina,
+.desno {
+  position: absolute;
+  background-color: white;
+}
+
+.first {
+  width: 20px;
+  height: 8px;
+  border-radius: 2px;
+  margin-top: 12.5px;
+  margin-left: 10px;
+  z-index: 100;
+}
+
+.lijevo {
+  width: 5px;
+  height: 12px;
+  border-radius: 2px;
+  margin-top: 16px;
+  margin-left: 4px;
+  z-index: 80;
+
+  transform: rotate(-20deg);
+}
+.sredina {
+  width: 28px;
+  height: 4px;
+  border-radius: 5px;
+  margin-top: 24px;
+  margin-left: 6px;
+  z-index: 100;
+}
+
+.desno {
+  width: 5px;
+  height: 12px;
+  border-radius: 2px;
+  margin-top: 16px;
+  margin-left: 31px;
+  z-index: 80;
+  transform: rotate(20deg);
+}
+
+.container {
+  position: relative;
+  border-radius: 5px;
+  background-color: #0057a3;
+  padding: 20px 0 30px 0;
+  margin: auto;
+  width: 50%;
+  border: 10px;
+  box-shadow: 10px 10px #c9f364;
+}
+
+input[type="submit"]:hover {
+  background-color: #c9f364;
+  color: #0057a3;
+}
+
+.input-container {
+  margin-bottom: 10px;
+  position: relative;
+  border-radius: 8px;
+  overflow: hidden;
+  height: 48px;
+  margin: 10px auto;
+  max-width: 400px;
+}
+/* @media screen and (min-width: 600px) {
+  .container {
+    background-color: violet !important;
+  }
+} */
+
+/* INPUT */
+
+.input-container label {
+  height: 20px;
+  position: absolute;
+  top: calc((100% - 20px) / 2);
+  z-index: 100;
```

Comment:
bolje bi bilo da se izbjegne `z-index` jer konceptualno zelimo da je label iza inputa jer ako je ispred onda blokira klik na input (raditi ce kada se doda id na input i for na label)